config/Dependencies.cmake

```diff
@@ -55,12 +55,12 @@ SET(LIBVLC_EXTRA_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/libvlc-headers/include)
 IF(NOT MOBILE)
     FIND_PACKAGE(LIBVLC REQUIRED)
 ENDIF()
-IF(MINGW OR MSVC)
-    STRING(REGEX REPLACE "\\\\" "/" LIBVLC_INCLUDE_DIR "${LIBVLC_INCLUDE_DIR}")
-    STRING(REGEX REPLACE "sdk/include" "" LIBVLC_BIN_DIR "${LIBVLC_INCLUDE_DIR}")
-ELSE()
-    STRING(REGEX REPLACE "include" "" LIBVLC_BIN_DIR "${LIBVLC_INCLUDE_DIR}")
-ENDIF()
+#IF(MINGW OR MSVC)
```

Comment:
This is not a valid fix. Completely remove the code or keep it in (preferred by me). Either way, remove this commit.